frontstage/views/secure_messaging.py

```diff
@@ -42,13 +44,25 @@
 def create_message(session):
     """Handles sending of new message"""
 
+    url = app.config['RAS_PARTY_GET_BY_RESPONDENT'].format(app.config['RAS_PARTY_SERVICE'], session['user_uuid'])
+    # url = "http://localhost:8050/api/party-api/respondents/id/" + session['user_uuid']
+    party_response = requests.get(url)
+    if party_response.status_code != 200:
+        return redirect(url_for('error_bp.error_page'))
+    party_response_json = party_response.json()
+    associations = party_response_json.get('associations')
+    if associations:
+        ru_id = associations[0].get('partyId')
+    else:
+        ru_id = None
+
```

Comment:
Moved it outside and now doing it in drafts as well as its an extra 2 lines of code